kuma/javascript/src/header/login.jsx

```diff
@@ -61,13 +63,29 @@ export default function Login(): React.Node {
             </div>
         );
     } else {
+        /**
+         * Send a signal to GA when a user clicks on the Sing In
+         * lnk in the header.
+         * @param {String} service - The service used to sign in
+         */
+        const sendSignInEvent = service => {
+            ga('send', {
+                hitType: 'event',
+                eventCategory: 'Authentication',
+                eventAction: 'Started sign-in',
+                eventLabel: service
+            });
+        };
         // Otherwise, show a login prompt
         return (
             <a
                 href={`/users/github/login/?next=${LOCATION}`}
                 data-service="GitHub"
                 rel="nofollow"
                 className="signin-link"
+                onClick={() => {
+                    sendSignInEvent('Github');
```

Comment:
Same point here. Instead of spelling `Github` twice (it's already misspelled) we can use `event.target.dataset.service` to get the same string.

Reply:
>  (it's already misspelled)

You mean because it is GitHub? That is based on their brand guidelines.

Reply:
Whoops, I made the 💩 💩 ;p